### ISSUE TITLE
file info iframe does not get enough height

### ISSUE THREAD

**Issue by fritzmg** [contao/core#8631]
If you view the info of a file in the file manager for a non-image file, the iframe won't always be high enough to view all infos without scrolling (depending on the length of the infos). Happens in Contao 3.5 and 4.3:

![screen shot 2017-01-23 at 10 43 37](https://cloud.githubusercontent.com/assets/4970961/22198953/17ed2ab2-e159-11e6-9206-1985b2054d1d.png)

![screen shot 2017-01-23 at 10 44 13](https://cloud.githubusercontent.com/assets/4970961/22198952/17ec4688-e159-11e6-852f-3b2e6f461c43.png)

It seems the iframe is only automatically resized when viewing the info of images.

**Comment by asaage** [comment]
I think thats because the modal tries to fit itself on the page.

in addition to that i get an anoying overflow-x scrollbar in chrome.

![image](https://cloud.githubusercontent.com/assets/4138336/22199977/6cb34faa-e15d-11e6-9e2f-6670012e5a8c.png)

simplemodal.min.css
```css
 .simple-modal .simple-modal-body div.contents {
    overflow-y: hidden;
    overflow-x: scroll;  /* should be auto */
}
```

**Comment by fritzmg** [comment]
> I think thats because the modal tries to fit itself on the page.

No, sorry, the screenshot was not quite clear on that. The screenshot is cropped. The height of the screenshot does not reflect the total height of the viewport.

**Comment by leofeyer** [comment]
That is because there are line breaks in the label and the path, which makes the content of the iframe higher than the calculated height. I don't really see a problem here.

**Comment by asaage** [comment]
overflow-x: scroll and line breaks... i think that does not fit together 😄

**Comment by fritzmg** [comment]
@leofeyer these aren't line breaks. Just regular wrapping. Also as you can see sometimes the left column is not wide enough, so `Zuletzt geändert am:` wraps to
```
Zuletzt geändert
am:
```
creating two lines as well, instead of just one line.

> I don't really see a problem here.

The problem is, that sometimes the iframe of the modal window is not high enough so view all its content, creating an unnecessary scroll bar.

**Comment by fritzmg** [comment]
Other examples:

![ss01](https://cloud.githubusercontent.com/assets/4970961/23742893/f86580a8-04ae-11e7-812b-88f3964c23a3.png)

![ss02](https://cloud.githubusercontent.com/assets/4970961/23742897/fc846e38-04ae-11e7-9c9f-6f3366ad1bd3.png)

The problem is, that the `height` parameter for the `Backend.openModalIframe` seems to be calculated by PHP - a better solution would be if the height was calculated according to the size of its content.

**Comment by leofeyer** [comment]
Fixed in e6aa1b82f6d40978dff582a54d317a0ccfb25be5.
